src/widgets/_windows.scss

```diff
@@ -117,6 +117,23 @@ window:not(.popup):not(.menu) {
             text-shadow: none;
             -gtk-icon-shadow: none;
         }
+
+        pane-separator {
```

Comment:
I don't think this is a valid selector. What is this intended to style?

Reply:
Oh this was found in some GNOME apps.